src/components/Tables/TableFilterBar.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import React from "react";
+
+interface Props {
+    filterText: { [key: string]: string };
+    onFilter: (e: React.ChangeEvent<HTMLInputElement>, filterField: string) => void; // eslint-disable-line no-unused-vars
```

Comment:
should be fixed by rebasing on more recent main - try and remove this comment and see? 🤞

Reply:
Rebased as suggested!